src/GitHub.VisualStudio/GitHub.VisualStudio.vsct

```diff
@@ -134,6 +145,12 @@
     <GuidSymbol name="guidManageConnections" value="{0A014553-A0AA-46DD-8D6B-B8E3178CA435}">
       <IDSymbol name="idManageConnections" value="0x1009"/>
     </GuidSymbol>
+
+    <!-- This is the guid used to group the menu commands together -->
+    <GuidSymbol name="guidCreateGistCommandPackageCmdSet" value="{D265E7EB-F3FC-411D-B4DC-1C6AD9264C4B}">
+        <IDSymbol name="MyMenuGroup" value="0x1020" />
```

Comment:
Probably want to give this id a slightly more descriptive name, for our future selves.